diff --git a/node_modules/i2c-bus/i2c-bus.js b/node_modules/i2c-bus/i2c-bus.js
index 3a685cf..d645ee5 100644
--- a/node_modules/i2c-bus/i2c-bus.js
+++ b/node_modules/i2c-bus/i2c-bus.js
@@ -1,7 +1,10 @@
 'use strict';

 var fs = require('fs'),
-  i2c = require('bindings')('i2c.node');
+  i2c = require('bindings')({
+    bindings: 'i2c.node',
+    module_root: __dirname,
+  });

 var DEVICE_PREFIX = '/dev/i2c-',
   FIRST_SCAN_ADDR = 0x03,
